x-pack/plugins/security_solution/common/api/asset_criticality/create_asset_criticality.schema.yaml

```
@@ -0,0 +1,23 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Create Asset Criticality Schema
```

Nit: just for consistency with the other routes:

```suggestion
  title: Asset Criticality Create Record Schema
```

---

Resolved in https://github.com/elastic/kibana/pull/172073/commits/109bfcf482a5d2bcfe24aa5c6fad598103c5359d